en/use-auto-failover.md

```diff
@@ -74,15 +101,20 @@ kubectl edit tc -n ${namespace} ${cluster_name}
 
 TiDB Operator will scale in the newly created Pods automatically. When the scaling in is finished, configure `spec.tikv.recoverFailover: false` to avoid the auto-scaling operation when the next failover occurs and recovers.
 
-### Failover with TiDB
+### Failover with TiFlash
 
-The TiDB automatic failover policy works the same way as `Deployment` does in Kubernetes. Assume that there are 3 nodes in a TiDB cluster. If a TiDB node is down for over 5 minutes (configurable by modifying `tidbFailoverPeriod`), TiDB Operator creates a new TiDB node. At this time, there are 4 nodes in the cluster. When the failed TiDB node gets back online, TiDB Operator deletes the newly created node and the number of nodes gets back to 3.
+TiDB Operator collects the TiFlash store health status by accessing the PD API and records the status in the `.status.tiflash.stores` field in TidbCluster CR.
 
-### Failover with TiFlash
+Take a TiFlash cluster with 3 Pods as an example. When a TiFlash Pod fails, the store status of the Pod changes to `Disconnected`. By default, after 30 minutes (configurable by changing `max-store-down-time = "30m"` in the `[schedule]` section of `pd.config`), the status changes to `Down`. Then, TiDB Operator automatically does the following operations:
+
+1. Wait for another 5 minutes (configurable by modifying `tiflashFailoverPeriod`), if the TiFlash Pod is still not recovered, TiDB Operator records the Pod information in the `.status.tiflash.failureStores` field of TidbCluster CR.
+2. When calculating the replicas of TiFlash StatefulSet, TiDB Operator takes the `.status.tiflash.failureStores` into account, so it will create a new Pod. Then, 4 Pods will exist at the same time.
 
-When a TiFlash Pod fails, its status turns to `Disconnected`. After 30 minutes (configurable by setting [`max-store-down-time`](https://pingcap.com/docs/stable/pd-configuration-file/#max-store-down-time) to `"30m"` in PD's configuration file), the status becomes `Down`. After waiting for another 5 minutes (configurable by modifying `tiflashFailoverPeriod`), if this TiFlash Pod is still down, TiDB Operator creates a new TiFlash Pod.
+When the failed Pod in the cluster recovers, TiDB Operator **DOES NOT** scale in the newly created Pod, but continues to keep 4 Pods. This is because scaling in TiFlash Pods will trigger data migration, which might affect the cluster performance.
```

Comment:
```suggestion
When the failed Pod in the cluster recovers, TiDB Operator **DOES NOT** remove the newly created Pod, but continues to keep 4 Pods. This is because scaling in TiFlash Pods will trigger data migration, which might affect the cluster performance.
```